src/it/scala-2.12/org/apache/james/gatling/ImapListMessagesBodyStructureIT.scala

```diff
@@ -0,0 +1,25 @@
+package org.apache.james.gatling
+
+import org.apache.james.gatling.imap.scenari.ImapListMessagesBodyStructureScenario
+import org.apache.james.gatling.jmap.MailboxName
+
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{Await, Future}
+
+
+class ImapListMessagesBodyStructureIT extends ImapIT {
+  private val MAILS_FETCHED = 1
+  private val mailboxes = List(MailboxName("INBOX"), MailboxName("rmbx0"), MailboxName("rmbx1.smbx5"), MailboxName("rmbx9"))
```

Comment:
what are these magic numbers?

Reply:
Magic numbers?

Reply:
what are these 0, 1.5, 9? where do they come from?

Reply:
I was trying to match a bit the provisioned mailboxes name in these fixtures

Reply:
if we restart the provisionning, will these names change?

Reply:
If we do not changed the provisioning strategy, no.

Anyway, we are in integration tests here, we just want to check that it works with arbitrary mailbox names.